files/justfiles/hardening.just

```diff
@@ -106,13 +106,14 @@ flatpak-permissions-lockdown:
 setup-usbguard:
     #!/usr/bin/bash
     echo "Notice: This will generate a policy based on your existing connected USB devices."
-    ACTIVE_USERNAME=$(whoami)
     run0 sh -c '
         mkdir -p /var/log/usbguard
         mkdir -p /etc/usbguard
         chmod 755 /etc/usbguard
         usbguard generate-policy > /etc/usbguard/rules.conf
+	sed -i "/IPCAllowedGroups=wheel/s/$/ usbguard/" /etc/usbguard/usbguard-daemon.conf
```

Comment:
@radioaddition justfile validation is failing because this is a tab :smile:

Reply:
wait like it is a tab and should be spaces or it should be a tab and isn't?

Reply:
when I open the editor they look the same ;-;